package.json

```diff
@@ -52,8 +52,9 @@
     "node": ">=16"
   },
   "dependencies": {
-    "coral": "git+ssh://git@github.com:Caleydo/coral#develop",
-    "ordino": "git+ssh://git@github.com:Caleydo/ordino#develop",
+    "canvas-confetti": "^1.4.0",
+    "coral": "git+ssh://git@github.com/Caleydo/coral#dev_d3_merge_fix",
```

Comment:
Should that not be the develop branch?

Reply:
yes, but it does not build when referencing develop

therefore the comment: https://github.com/Caleydo/coral_public/pull/15#issuecomment-1266489192 ;-)